src/components/Navbar/Navbar.tsx

```diff
@@ -13,34 +13,18 @@ const Navbar = () => {
     setRole('');
   };
 
-  const renderAdminLinks = () => {
-    return (
-      <>
-        <Link to="/dashboard" className={classes['c-navbar__link']}>
-          Overview
-        </Link>
-        <Link to="/users" className={classes['c-navbar__link']}>
-          Users
-        </Link>
-      </>
-    );
-  };
-
   return (
     <div className={classes['c-navbar']}>
       <div>
-        <Link to={`/booksoverview`} className={classes['c-navbar__link']}>
+        <Link to={`/`} className={classes['c-navbar__link']}>
```

Comment:
There is no need to use curly brackets when you pass a string to an attribute to, replace this with `to="/"`